### PR TITLE
If registry_url != registry.access.redhat.com then modify image streams

### PR DESCRIPTION
**Pull request by sdodson**

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
@abutcher ptal

**Comment by abutcher** [comment]
Looks okay to me -- @detiber ptal

File: roles/openshift_examples/tasks/main.yml [hunk 1]
```diff
@@ -4,6 +4,11 @@
     src: "examples/{{ content_version }}/"
     dest: "{{ examples_base }}/"
 
+- name: Modify registry paths if registry_url is not registry.access.redhat.com
+  shell: >
+    find {{ examples_base }} -type f | xargs -n 1 sed -i 's|registry.access.redhat.com|{{ registry_host }}|g'
+  when: "{{ registry_host != '' }}"
```
Comment (abutcher) [inline roles/openshift_examples/tasks/main.yml]: Can untemplate this.

**Comment by detiber** [comment]
aos-ci-test

File: roles/openshift_examples/tasks/main.yml [hunk 2]
```diff
@@ -4,6 +4,11 @@
     src: "examples/{{ content_version }}/"
     dest: "{{ examples_base }}/"
 
+- name: Modify registry paths if registry_url is not registry.access.redhat.com
+  shell: >
+    find {{ examples_base }} -type f | xargs -n 1 sed -i 's|registry.access.redhat.com|{{ registry_host }}|g'
```
Comment (detiber) [inline roles/openshift_examples/tasks/main.yml]: Since registry_host could be user provided, we should probably use the `quote` filter here.

**Comment by maci0** [comment]
something like this for the templates would also make sense to replace github.com by an enterprise's internal git server that is hosting the same examples

**Comment by etsauer** [comment]
@sdodson @abutcher what's the status of this?

**Comment by abutcher** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Pull request merged**